.github/workflows/ci.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Install Rust Stable
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.51.0 # Pinned warnings
+          toolchain: 1.56.1 # Pinned warnings
```

Comment:
Any particular reason to not bump to latest? I don't know best practices for Rust versions.

Reply:
https://github.com/pest-parser/pest/pull/615#issuecomment-1179755352

Reply:
It's meant as a sort of "LTS".